Resources/Prototypes/Catalog/Fills/Lockers/heads.yml

```diff
@@ -118,8 +124,14 @@
       - id: BoxEncryptionKeyPassenger
       - id: BoxEncryptionKeyService
       - id: ClothingBackpackIan
-        prob: 0.5
+#        prob: 0.5
       - id: AccessConfigurator
+      - id: MedalCase
+      - id: JetpackCaptainFilled
+      - id: ClothingHandsGlovesCaptain
+      - id: ClothingOuterHardsuitCap
+      - id: ClothingMaskGasCaptain
+      - id: ClothingBeltSheathFilled
```

Comment:
also we dont really want to mix drip either, SR walking around looking like a captain is going to confuse people. They should get either a generic or a special Station Rep hard suit, gas mask, gloves, etc.

Reply:
Temped removed till we can add new sprites,